controls/combobox/commands.md

```diff
@@ -12,9 +12,17 @@ ComboBox has the following commands:
 
 - `SelectAllCommand` (`ICommand`)&mdash;Selects all items from the source.
 
-> SelectAll command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown, if the command is invoked in `Single` `SelectionMode`.
+> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown if the command is invoked in `Single` `SelectionMode`.
 
-- `ClearSelectionCommand` (*ICommand*)&mdash;Sets the selection to null. If Multiple SelectionMode is used, this command will clear all selected items.
+- `ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If Multiple `SelectionMode` is used, this command will clear all selected items.
+
+Here is an example with `ClearSelectionCommand`:
```

Comment:
Add more information that you can override the default behavior and create custom command, like in the example. Also explain that the example shows both states, default execution and custom command execution